.github/semantic.yml

```diff
@@ -0,0 +1,5 @@
+# Config file for Semantic Pull Requests Probot
+# See https://github.com/probot/semantic-pull-requests
+
+# Always validate the PR title, and ignore the commits
```

Comment:
ignore the commit message body?

Reply:
I think it will take a little playing with to figure out the best configuration: this is what components uses so i figured we start there.

Because we squash merge, the PR title is the most important part since the changelog entry is generated only from that one line pr/commit title.

The conventional commit spec allows for footers in the body but I think the only footer we'd care about is `BREAKING CHANGES` and for that we should use the `!` convention. E.g. `feat!: removed legacy public property on foo`.